contextstorage/build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id "java-library"
+    id "maven-publish"
```

Comment:
Let's remove maven-publish until we are confident we like the name. With this removed, we don't have to worry about accidentally publishing it.